language-server/lsp/proofState.ml

```diff
@@ -11,12 +11,31 @@
 (*   See LICENSE file.                                                    *)
 (*                                                                        *)
 (**************************************************************************)
+type hyp = {
+  identifiers: string list;
+  type_ : string;
```

Comment:
@maximedenes is this a typo ? Or is it meant to be called type_ ? It currently breaks hypothesis display in the goal view: where should I change it, in the language-server or in the client ?